part3/intro.rst

```diff
@@ -7,7 +7,7 @@ gem5 Full System Simulation
 ----------------------------
 
 One of the most exciting features of gem5 is the ability to simulate the full system.
-In full system mode, gem5 acts more like an emulator or hypervisor than a traditional simulator.
+In syscall emulation mode, gem5 acts more like an emulator or hypervisor than a traditional simulator.
```

Comment:
I meant full system here. Maybe it would be more clear with:

In FS mode, gem5 acts more like an emulator (e.g., qemu) or a hypervisor (e.g., VMWare and Xen) than a traditional simulator.

Reply:
I see.  I thought you were trying to compare the two modes.  Anyways, feel free to fix as you see fit.

It would have been funnier if you spelled implementation wrong above :p.  I don't think any of these fixes are spelling fixes though.  Regardless, no problem at all.

Matt